examples/nomad-consul-ami/nomad-consul.json

```diff
@@ -6,72 +6,92 @@
+        ],
+        "most_recent": true
       },
-      "owners": ["099720109477"],
-      "most_recent": true
+      "ssh_username": "ubuntu"
     },
-    "ssh_username": "ubuntu"
-  },{
-    "ami_name": "nomad-consul-amazon-linux-{{isotime | clean_ami_name}}",
-    "ami_description": "An Amazon Linux AMI that has Nomad and Consul installed.",
-    "instance_type": "t2.micro",
-    "name": "amazon-linux-ami",
-    "region": "{{user `aws_region`}}",
-    "type": "amazon-ebs",
-    "source_ami_filter": {
-      "filters": {
-        "virtualization-type": "hvm",
-        "architecture": "x86_64",
-        "name": "*amzn-ami-hvm-*",
-        "block-device-mapping.volume-type": "gp2",
-        "root-device-type": "ebs"
+    {
+      "ami_name": "nomad-consul-amazon-linux-{{isotime | clean_ami_name}}",
+      "ami_description": "An Amazon Linux AMI that has Nomad and Consul installed.",
+      "instance_type": "t2.micro",
+      "name": "amazon-linux-ami",
+      "region": "{{user `aws_region`}}",
+      "type": "amazon-ebs",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "architecture": "x86_64",
+          "name": "*amzn-ami-hvm-*",
+          "block-device-mapping.volume-type": "gp2",
+          "root-device-type": "ebs"
+        },
+        "owners": [
+          "amazon"
+        ],
+        "most_recent": true
       },
-      "owners": ["amazon"],
-      "most_recent": true
+      "ssh_username": "ec2-user"
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "shell",
+      "inline": [
+        "sudo apt-get install -y git"
+      ],
+      "only": [
+        "ubuntu16-ami"
+      ]
     },
-    "ssh_username": "ec2-user"
-  }],
-  "provisioners": [{
-    "type": "file",
-    "source": "{{template_dir}}/../../../terraform-aws-nomad",
-    "destination": "/tmp",
-    "pause_before": "30s"
-  },{
-    "type": "shell",
-    "inline": [
-      "/tmp/terraform-aws-nomad/modules/install-nomad/install-nomad --version {{user `nomad_version`}}"
-    ]
-  },{
-    "type": "shell",
-    "inline": [
-      "sudo apt-get install -y git"
-    ],
-    "only": ["ubuntu16-ami"]
-  },{
-    "type": "shell",
-    "inline": [
-      "sudo yum install -y git"
-    ],
-    "only": ["amazon-linux-ami"]
-  },{
-    "type": "shell",
-    "inline": [
-      "git clone --branch {{user `consul_module_version`}} https://github.com/hashicorp/terraform-aws-consul.git /tmp/terraform-aws-consul",
-      "/tmp/terraform-aws-consul/modules/install-consul/install-consul --version {{user `consul_version`}}"
-    ]
-  }]
+    {
+      "type": "shell",
+      "inline": [
+        "sudo yum install -y git"
+      ],
+      "only": [
+        "amazon-linux-ami"
+      ]
+    },
+    {
+      "type": "file",
+      "source": "{{template_dir}}/../../../terraform-aws-nomad",
+      "destination": "/tmp",
+      "pause_before": "30s"
+    },
+    {
+      "type": "file",
+      "source": "setup_nomad_consul.sh",
```

Comment:
Why in this Packer template are you copying the script to the server and running it from `/tmp`, whereas in the above you are just executing it directly using the `script` keyword? We should stick with one approach for consistency and it seems like the latter is simpler.